docs/config.md

```diff
@@ -15,6 +15,8 @@ services:
     type: redis:5
     persist: false
     portforward: false
+    creds:
```

Comment:
@reynoldsalec this `creds.password` key seems to be at at odds with the `password` key used below. should it be an empty string by default or `false`?